src/style/_fonts.ts

```diff
@@ -1,4 +1,5 @@
 export const fontFamilyString = [
+  'Rubik',
   '"Alef"',
```

Comment:
Do we still need Alef font?
If not, remove here and the import in index.html

Reply:
I wasn't sure about that so I left it there , I'll ask Yuval about it , thanks .